infra/event-sync/src/main/java/com/simprints/infra/eventsync/sync/down/tasks/EventDownSyncTask.kt

```diff
@@ -75,7 +75,7 @@ internal class EventDownSyncTask @Inject constructor(
                 .catch {
                     // Track a case when event stream is closed due to a parser error,
                     // but the exception is handled gracefully and channel is closed correctly.
-                    errorType = it.toString()
+                    errorType = it.javaClass.simpleName
```

Comment:
Maybe you can add a test to prevent regression if someone updates this code (and same for the EventUpSyncTask)

Reply:
@ybourgery Makes sense, added.